galleries/examples/subplots_axes_and_figures/invert_axes.py

```diff
@@ -1,25 +1,36 @@
 """
-===========
-Invert Axes
-===========
+=============
+Inverted axis
+=============
 
-You can use decreasing axes by flipping the normal order of the axis
-limits
+There are two ways to invert the direction of an axis:
```

Comment:
```suggestion
This example demonstrates two ways to invert the direction of an axis:
```
My turn to be pedantic!  There are [many ways](https://stackoverflow.com/questions/2051744/how-to-invert-the-x-or-y-axis/78298718#78298718), but I think only these two are worth highlighting.